ui/lib/core/icon-mappings.js

```diff
@@ -0,0 +1,218 @@
+export const localIconMap = {
```

Comment:
Could we get a couple comments for our future selves about what the mappings mean? Eg. "if a value is null in `localIconMap` it means there isn't a Flight Icon equivalent"

Reply:
Good call I added some in the file.